ext/pcntl/tests/pcntl_signal_002.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+pcntl_signal() - If handler is an int value different than SIG_DFL or SIG_IGN
+--EXTENSIONS--
+pcntl
+--FILE--
+<?php
+
+try {
+    pcntl_signal(SIGTERM, (SIG_DFL - SIG_IGN - 1));
```

Comment:
What if `SIG_DFL == 3` and `SIG_IGN == 1`? In this case we would pass `1` which would be `SIG_IGN`. To my knowledge, the only guarantee about `SIG_DFL` and `SIG_IGN` is that their values are not equal to an address of any function, so passing `0` should be fine.

Reply:
I will adjust

Reply:
Tried with 0 and did not work. 
I tried -1 and works. 

Dirty solution but works.